### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/01/day0.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#3] [Comments]
# https://mnfienen.github.io/talulat/2023/08/01/day0.html

It’s Day 0 - from the beach in SW Michigan to Sarnia, Ontario CANADA miles today: 0 total miles: 0 km today: 0 total km: 0

https://mnfienen.github.io/talulat/2023/08/01/day0.html

**Comment by keikomiller** [comment]
Woo hoo!!  We miss you at the beach but wish you nothing but tailwinds!
